src/java/org/apache/cassandra/schema/MigrationCoordinator.java

```diff
@@ -180,6 +199,18 @@ boolean wasReceived()
         {
             return receivedSchema;
         }
+
+        @Override
+        public String toString()
+        {
+            return "VersionInfo{" +
+                   "version=" + version +
+                   ", outstandingRequests=" + outstandingRequests +
+                   ", requestQueue=" + requestQueue +
+                   ", waitQueue=" + waitQueue.getWaiting() +
```

Comment:
Maybe it should be `waitQueueSize`?
```suggestion
                   ", waitQueueSize=" + waitQueue.getWaiting() +
```